python/ray/data/_internal/planner/random_shuffle.py

```diff
@@ -1,11 +1,14 @@
 import time
-from typing import Any, Dict, List, Optional, Tuple
+from typing import Any, Dict, List, Optional
 
 from ray.data._internal.execution.interfaces import (
     AllToAllTransformFn,
     RefBundle,
     TaskContext,
 )
+from ray.data._internal.execution.interfaces.transform_fn import (
+    AllToAllTransformFnResult,
+)
```

Comment:
Appreciate the cleaning up that you're doing but let's avoid doing unrelated changes to already huge PR next time